sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/AppNameUtil.java

```diff
@@ -60,34 +60,15 @@ private AppNameUtil() {
         RecordLog.info("App name resolved: " + appName);
     }
 
+    /**
+     * resolve app name from jvm options
+     */
     public static void resolveAppName() {
```

Comment:
We have to consider the case when `project.name` is absent both in system property and configuration file, so we have to provide a default parsed appName rather than empty.